loc/factions/vanguard.js

```diff
@@ -8,7 +8,7 @@ export default {
     en: 'The Vanguard horde seeks tradition, structure and order in society. Their castle can be found in a lush green area around Guardstone.',
     sv: 'Vanguard-horden eftersträvar ett ordnat och traditionsdrivet samhälle. Deras slott ligger i ett frodigt område kring Guardstone.',
     ru: 'Орда Vanguard ищет традиции, структуру и порядок в обществе. Их крепость может быть найдена в пышной зеленой зоне около Guardstone.',
-    es: 'Los valores del Vanguard son la tradición, la estructura y el orden de sociedad. Su Castillo se halla en el bosque exuberante de Guardstone.',
+    es: 'Los valores de la Vanguardia son la tradición, la estructura y el orden de sociedad. Su Castillo se halla en el bosque exuberante de Guardstone.',
```

Comment:
why did you switch it to Vanguardia? We use Vanguard through all the translations for clarity. Please dont randomly change things without confirming with someone whether it is intended.

Reply:
ok dek i will correct it